tests/scipy_stats_test.py

```diff
@@ -322,6 +322,12 @@ def args_maker():
                             tol=1e-3)
     self._CompileAndCheck(lax_fun, args_maker)
 
+  def testIssue10219(self):
```

Comment:
We've stopped naming tests this way because they're less descriptive than we'd like them to be. Can you do something like this instead? https://github.com/google/jax/blob/902fc0c3d2b3ec9b6034c66074984386ec35606f/tests/lax_numpy_test.py#L2081-L2082

Reply:
Done.